### PR TITLE
update: version & add: fast-flags

### PR DESCRIPTION
**Pull request by anshalshukla**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -13,11 +13,25 @@ struct Cli {
...
     /// Init a new App Chain config
-    Init,
+    Init {
+        /// App chain name
+        #[arg(short, long = "chain-name")]
+        name: Option<String>,
+        /// Select Sovereign
+        #[arg(short, long = "chain-mode")]
+        mode: Option<String>,
+        /// Select between Avail, Celestia, Ethereum, NoDA
```
Comment (apoorvsadana) [inline src/main.rs:23]: let the comment be select da layer, otherwise you need to change on every new addition

File: src/main.rs [hunk 2]
```diff
@@ -13,11 +13,25 @@ struct Cli {
 #[derive(Subcommand)]
 enum Commands {
     /// Init a new App Chain config
-    Init,
+    Init {
+        /// App chain name
+        #[arg(short, long = "chain-name")]
+        name: Option<String>,
+        /// Select Sovereign
```
Comment (apoorvsadana) [inline src/main.rs:20]: let the comment be select DaMode

Comment (anshalshukla) [reply src/main.rs:20]: Same as above, it should be exact string

File: src/main.rs (hunk 1 above)
Comment (anshalshukla) [reply src/main.rs]: found a way to pass Enum itself, updated

File: src/utils/madara.rs [hunk 3]
```diff
@@ -60,6 +60,8 @@ pub fn setup_and_run_madara(config: AppChainConfig) -> Result<(), MadaraError> {
         &base_path,
     ];
 
+    args.extend(flags.split(",").map(|flag| flag.trim()).filter(|flag| !flag.is_empty()));
```
[the remark this answers is not in the document]
Comment (anshalshukla) [reply src/utils/madara.rs]: updated

File: src/cli/explorer.rs [hunk 4]
```diff
@@ -48,12 +48,6 @@ pub async fn explorer(opts: &ExplorerOpts) {
...
     }
 
-    run_docker_image(
-        "ghcr.io/lambdaclass/stark_compass_explorer:v0.2.34.3-1",
-        CONTAINER_NAME,
-        Some(env),
-        Some(host_config),
-    )
-    .await;
+    run_docker_image("anshalshukla/starkcompass:latest", CONTAINER_NAME, Some(env), Some(host_config)).await;
```
Comment (apoorvsadana) [inline src/cli/explorer.rs:51]: let's ask them to create a release? or let's push it on our ECR if you think that will take time

**Pull request merged**